### PR TITLE
[Do not merge] Deploy in k8s

### PR DESCRIPTION
**Pull request by smeruelo**
Please don't merge until a migration plan is discussed.

### CODE REVIEW

File: helm/README.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+This is the [Helm](https://helm.sh/) chart to deploy [source{d} blog](https://blog.sourced.tech) in kubernetes.
+
+# Introduction
+
+The blog website is generated using [Hugo](http://gohugo.io/)
+and served with [Caddy](https://caddyserver.com/) web server.
+A docker container is built by our self hosted [Drone](https://drone2.srcd) server,
+and deployed using this [helm plugin](https://github.com/ipedrazas/drone-helm) in our GKE kubernetes clusters.
+
```
Comment (rporres) [inline helm/README.md]: Although interesting, Drone is not related to this chart. I would place this information in a different place.

File: helm/README.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+Pushes to master branch in the [repository](https://github.com/src-d/blog), trigger a drone job.
+A new staging docker image is built and uploaded to [Quay registry](https://quay.io/repository/srcd/blog).
+It is labeled with the commit hashtag.
+The blog pods running on our k8s _staging_ cluster are then upgraded to use this new image.
+
+If a tag in the form is pushed along with the master branch HEAD,
+then a new production docker image is built too,
+and uploaded and labeled in an analog way.
+Then the pods running on our k8s _production_ cluster are upgraded to use this new image.
+
```
Comment (rporres) [inline helm/README.md]: This Workflow is not related to the chart. This information should go somewhere else.

File: helm/README.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+The most relevant parameters are:
+
+| Parameter              | Description                                   | Default                     |
+| ---------------------- | --------------------------------------------- | --------------------------- |
+| image.tag              | Label to identify image in quay               | none (it's mandatory)       |
+| ingress.staticIP       | Static IP (GCE resource's name)               | none (it's mandatory)       |
+| ingress.hostname       | Blog's DNS                                    | none (it's mandatory)       |
+| replicaCount           | Number of pods                                | `2`                         |
+| image.repository       | Path to docker registry                       | `quay.io/srcd/blog`         |
+| kubernetes.io/tls-acme | If certificate should be handled by kube-lego | `true`                      |
```
Comment (rporres) [inline helm/README.md]: you cannot set this annotation by itself without setting the rest

File: helm/README.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+Other parameters can be provided too but, if not, a default value will be used.
+
+# Configuration
+
+The most relevant parameters are:
+
+| Parameter              | Description                                   | Default                     |
+| ---------------------- | --------------------------------------------- | --------------------------- |
+| image.tag              | Label to identify image in quay               | none (it's mandatory)       |
+| ingress.staticIP       | Static IP (GCE resource's name)               | none (it's mandatory)       |
```
Comment (rporres) [inline helm/README.md]: I would call this globalStaticIpName to make clearer what it means

File: helm/templates/ingress.yaml [hunk 5]
```diff
@@ -0,0 +1,32 @@
+{{- if .Values.ingress.enabled -}}
```
Comment (rporres) [inline helm/templates/ingress.yaml]: Does it make sense to deploy this chart without ingress? If not, I would remove this logic.

File: helm/README.md (hunk 3 above)
Comment (smeruelo) [inline helm/README.md]: I guess not, but what are you suggesting?

File: helm/templates/ingress.yaml (hunk 5 above)
Comment (smeruelo) [inline helm/templates/ingress.yaml]: My criteria has been to differ as little as possible from the skeleton provided by `helm create` but, if this is how we prefer it, I guess more things could/should be changed.

File: helm/README.md (hunk 3 above)
Comment (rporres) [inline helm/README.md]: I'd just remove it from the README.md

**Comment by smeruelo** [comment]
Unlike the landing project, this one is currently being deployed using the content in this repository. In this PR I'm modifying the `Caddyfile`, that's why we should avoid merging this into master until we are prepared to migrate.

File: helm/README.md (hunk 1 above)
Comment (smeruelo) [inline helm/README.md]: Maybe in a "deployment" section in https://github.com/src-d/blog/blob/master/README.md? What do you think?

File: helm/templates/ingress.yaml (hunk 5 above)
Comment (rporres) [inline helm/templates/ingress.yaml]: The main structure of the skeleton is kept (services names, etc). I agree about not changing too much overall structure, but within reason:  there's no way this chart is deployed without an ingress hence we don't need this.

Having said that, if you want to keep it, it causes no harm for the moment :smile:

**Comment by rporres** [comment]
@smeruelo just tell me here when you're done pushing changes and I will review this again. Thx!

**Comment by smeruelo** [comment]
I'm done, you can review it when it suits you best. Thanks!

File: helm-charts/blog/templates/NOTES.txt [hunk 6]
```diff
@@ -0,0 +1,11 @@
...
+Access the deployed blog in the following URL:
+
+{{- if eq true .Values.ingress.tls }}
+      https://{{ .Values.ingress.hostname}}
+{{- else }}
+      http://{{ .Values.ingress.hostname}}
+{{- end }}
+
+NOTE: It may take a few minutes for the Ingress IP to be available.
+      Watch the status with: 'kubectl get ingress {{ template "fullname" . }}'
```
Comment (rporres) [inline helm-charts/blog/templates/NOTES.txt]: you need to use `describe` rather than `get` to see the status with all the needed detail

**Comment by smeruelo** [comment]
Last change added and ready to ship! @rporres @dpordomingo
